src/languages/en.js

```diff
@@ -532,6 +532,7 @@ export default {
         buttonConfirm: 'Got it',
         error: {
             noBankAccountAvailable: 'Sorry, no bank account is available',
+            noBankAccountSelected: 'Please choose an account',
```

Comment:
I redacted  `Please choose an account` for `selectedBank` Picker based on same Picker's label: `Choose an account` on `bankAccount.chooseAnAccount`

Reply:
Sorry, what is the need for this?

Reply:
I don't see an error text on `bankAccount.error` that I could use for [Picker's `selectedBank`](https://github.com/Expensify/App/pull/8765/files#diff-2a35d342f7be1cc5c1a8f02937f635a5c7083ec66b64b400027d65087d400e09L256-R257) when user submits and this Picker is still empty. 

If I don't provide an error text, the Picker will show no error.

if `noBankAccountSelected: 'Please choose an account'` is not set on `en.js` but [this change (new line 121)](https://github.com/Expensify/App/pull/8765/files#diff-2a35d342f7be1cc5c1a8f02937f635a5c7083ec66b64b400027d65087d400e09R121) is set, the Picker's error will show something like this:
![Screen Shot 2022-05-10 at 12 13 17 PM](https://user-images.githubusercontent.com/48106652/167706667-b43c07b3-c98e-43a9-b400-6054659e94d0.png)